### ISSUE TITLE
Can't install h264ify

### ISSUE THREAD

**Issue by Jorge9810** [muslayev/icecat-win64#5]
The .gif is self-explanatory. I'm using IceCat v60.4.0 and it just doesn't want to install the addon "h264ify".
![untitled](https://user-images.githubusercontent.com/43153921/52518546-c4575700-2c43-11e9-9bab-b6d781d893ad.gif)

**Comment by muslayev** [comment]
Hello. From what i gather, you want to disable webm v8? I don't know how to fix it (I think, it because of linux-specific patches for video playbacking). But manually you can switch it off:
about:config
media.webm.enabled = true

And check it at https://www.youtube.com/html5

**Comment by bogachenko** [comment]
@muslayev ты русифицированную версию используешь ? у меня такая хуйня постоянно происходила, когда я скачивал с левого сайта (я хотел русфикатор скачать с их сайта, под icecat, а не официальный мозиловский), а не с АМО. Отключение проверки расширений и добавление сайта в исключении в форме "Разрешенные сайты - Установка дополнений" ничего не дала. Я плюнул.

**Comment by muslayev** [comment]
> @muslayev ты русифицированную версию используешь ? или даже не так, ты вообще можешь хоть, что-нибудь накатить с их сайта, а не с АМО?...
> 
> у меня такая хуйня постоянно происходила, когда я скачивал с левого сайта (я хотел русфикатор скачать с их сайта, под icecat, а не официальный мозиловский), а не с АМО. Отключение проверки расширений и добавление сайта в исключении в форме "Разрешенные сайты - Установка дополнений" ничего не дала. Я плюнул.

Привет, не пользуюсь. Но я только что установил расширение здесь: https://ftp.gnu.org/gnu/gnuzilla/60.3.0/langpacks/
И добавил параметр в about:config: intl.locale.requested = ru-RU

**Comment by bogachenko** [comment]
точно помню что это был не  [https://ftp.gnu.org/](https://ftp.gnu.org/gnu/gnuzilla/60.3.0/langpacks/)

там был список типа таблицы в 3 ряда.. интерфейс сайта напоминал wikipedia.там было все на английском... сейчас этот сайт найти не могу... ~врядли это был васянский сайт? пиздец до чего докатился..~

ps с ftp русификатор устновился

**Comment by bogachenko** [comment]
@muslayev кстати а почему там только 60.3.0 это же офф сайт и его ftp, где там новые версии для 60.4.0 u 60.5.0. Где ты берешь новые версии? объясни

**Comment by muslayev** [comment]
> @muslayev кстати, а почему там только 60.3.0? это же офф сайт и его ftp, где там новые версии для 60.4.0 u 60.5.0. Где ты берешь новые версии? объясни

Здесь федоровцы под каждый релиз обновляют патчи https://src.fedoraproject.org/rpms/icecat/commits/master
А когда выходит релиз от гнузиллы, они тянут оттуда изменения.
По приоритету, я собираю релиз от гнузиллы.

**Comment by Jorge9810** [comment]
> From what i gather, are you going to disable webm v8?

No. I just want to force Youtube to use h264 instead of vp9, @muslayev. I tried things like setting `media.benchmark.vp9.threshold` to absurdly high values like 9001 in the past like certain posts on Reddit suggest, but while Youtube does display in the statistics of a video that it's indeed using the h264 encoder, the video usually lags a little bit, and that's probably because it's still using the vp9 encoder.
For me, h264ify is a vital addon.

One thing I'd like to know is why does it not work with these builds of IceCat for Windows, even though it supposedly works on Linux, as the Free Software Directory indicates.

**Comment by muslayev** [comment]
> > From what i gather, are you going to disable webm v8?
> 
> No. I just want to force Youtube to use h264 instead of vp9, @muslayev. I tried things like setting `media.benchmark.vp9.threshold` to absurdly high values like 9001 in the past like certain posts on Reddit suggest, but while Youtube does display in the statistics of a video that it's indeed using the h264 encoder, the video usually lags a little bit, and that's probably because it's still using the vp9 encoder.
> For me, h264ify is a vital addon.
> 
> One thing I'd like to know is why does it not work with these builds of IceCat for Windows, even though it supposedly works on Linux, as the Free Software Directory indicates.
> 
> **EDIT**: I forgot to mention that this same thing happens when trying to install certain themes to the browser too, [like this one](https://addons.mozilla.org/en-US/firefox/addon/t-ara-pink-k-pop/?src=search).
> ![icecat-2019-02-10_03-39-19](https://user-images.githubusercontent.com/43153921/52530519-b65a1280-2ce5-11e9-9051-c09c2acfbcc1.png)

About theme extensions, i found [here](http://boards.4channel.org/g/thread/69723718#p69727703), that someone have been faced with the similar issue on Linux.
>it supposedly works on Linux, as the Free Software Directory indicates.
No, i don't think so. Of course, FSF indicated it, long before Quantum builds were released by GNUZilla (accourding to date of posting). Additionally i checked - extension works with original ESR.
Anyway i will check it with Icecat on a Linux machine in the next week.
